archetypes/oci-mp/pom.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
shouldn't these copyright banners be removed?

Reply:
Copyright is removed from all files which are generated for end user but not from internal files that enables that generation.